components/server/src/auth/host-context-provider-impl.ts

```diff
@@ -58,9 +53,17 @@ export class HostContextProviderImpl implements HostContextProvider {
                 span.finish();
             }
         }, 1999);
+        this.disposables.push(updateDynamicHostsJob);
```

Comment:
@geropl, wouldn't that already do the initial `await this.updateDynamicHosts` as well? 
So, we can remove the L.40 call, no?